mitmproxy/filt.py

```diff
@@ -193,12 +193,22 @@ class FBod(_Rex):
     help = "Body"
 
     def __call__(self, f):
-        if f.request and f.request.content:
-            if self.re.search(f.request.get_decoded_content()):
-                return True
-        if f.response and f.response.content:
-            if self.re.search(f.response.get_decoded_content()):
-                return True
+
+        # HTTPFlow
+        if hasattr(f, 'request'):
```

Comment:
We could also use:

```python
from mitmproxy.models.http import HTTPFlow
from mitmproxy.models.tcp import TCPFlow

if isinstance(f, HTTPFlow):
    pass
elif isinstance(f, TCPFlow)
    pass
```

Is there a better way to distinguish between HTTPFlow and TCPFlow?